target/xilinx/constraints/vcu118.xdc

```diff
@@ -0,0 +1,85 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Nicole Narr <narrn@student.ethz.ch>
+# Christopher Reinwardt <creinwar@student.ethz.ch>
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+# Paul Scheffler <paulsc@iis.ee.ethz.ch>
+# Yvan Tortorella <yvan.tortorella@fondazione-chipsit.it>
+
+#############
+# Sys Clock #
+#############
+
+# 250 MHz input clock
+set SYS_TCK 4
+create_clock -period $SYS_TCK -name sys_clk [get_ports sys_clk_p]
+
+# SoC clock is generated by clock wizard and its constraints
+set SOC_TCK 20.0
+set soc_clk [get_clocks -of_objects [get_pins i_clkwiz/clk_50]]
+set_property CLOCK_DEDICATED_ROUTE BACKBONE [get_nets soc_clk]
+
+#######
+# MIG #
+#######
+
+# Dram axi clock : 300 MHz (defined by MIG constraints)
+set MIG_TCK 3
```

Comment:
Same as above

Reply:
This I think is identical to the VCU128 right?